draft-ietf-quic-transport.md

```diff
@@ -3793,8 +3796,8 @@ Issue and pull request numbers are listed with a leading octothorp.
 
 - Employ variable-length integer encodings throughout (#595)
 - Draining period can terminate early (#869)
-- Replace PING and PONG frames with PATH_CHALLENGE and PATH_RESPONSE frames,
-  clarify connection migration text (#000)
+- Added PATH_CHALLENGE and PATH_RESPONSE frames, removed PING with Data,
+  and rewrote connection migration (#000)
```

Comment:
Probably worth mentioning that we also removed the PONG frame

Reply:
good point, done.